lib/manager/gradle-wrapper/extract.ts

```diff
@@ -10,7 +10,7 @@ export function extractPackageFile(fileContent: string): PackageFile | null {
 
   let lineNumber = 0;
   for (const line of lines) {
-    const match = /^distributionUrl=.*-((\d|\.)+)-(bin|all)\.zip\s*$/.exec(
+    const match = /^distributionUrl\s*=\s*\S*-((\d|\.)+)-(bin|all)\.zip\s*$/.exec(
```

Comment:
@rarkins @JamieMagee We should move the regexes to module scope, otherwise they are recreated on every loop iteration, which is unnessessary